src/components/main/menu/Submit.jsx

```diff
@@ -69,7 +69,7 @@ const Submit = ({
         }
         break;
       }
-      default: return null;
+      default: return undefined;
```

Comment:
Changed to return undefined because react devtools was warning that effect functions shouldn't return null.